overlord/snapstate/snapmgr.go

```diff
@@ -278,6 +282,9 @@ func Manager(st *state.State) (*SnapManager, error) {
 	if err := os.MkdirAll(dirs.SnapCookieDir, 0700); err != nil {
 		return nil, fmt.Errorf("cannot create directory %q: %v", dirs.SnapCookieDir, err)
 	}
+	if err := os.MkdirAll(dirs.SnapCacheDir, 0755); err != nil {
```

Comment:
Please consider moving this below.

Reply:
Below what?

Reply:
below, where his other comment is (or was). I know what he means (and i forgot to do it in my last round)